lib/web_ui/lib/src/engine/semantics/focusable.dart

```diff
@@ -38,7 +38,14 @@ class Focusable extends RoleManager {
 
   @override
   void update() {
-    _focusManager.changeFocus(semanticsObject.hasFocus && (!semanticsObject.hasEnabledState || semanticsObject.isEnabled));
+    if (semanticsObject.isFocusable) {
+      if (!_focusManager.isManaging) {
+        _focusManager.manage(semanticsObject.id, semanticsObject.element);
+      }
+      _focusManager.changeFocus(semanticsObject.hasFocus && (!semanticsObject.hasEnabledState || semanticsObject.isEnabled));
+    } else {
+      _focusManager.stopManaging();
```

Comment:
Do we need to `changeFocus(false)` when the node becomes unfocusable?

Reply:
Good point. I also noticed that I unconditionally start managing the node in the constructor. Fixing.

Reply:
Done.